npc/quests/the_sign_quest.txt

```diff
@@ -4484,9 +4484,8 @@ cmd_in02,88,51,4	script	Strange Guy#sign	1_M_SIGNART,{
 				mes "I'll freakin' beat you to near";
 				mes "freakin' death! Bam bam bam!";
 				close2;
-				percentheal -100,0;
 				soundeffect "effect\\sign_noise.wav",1;
-				percentheal -99,0;
+				Hp -= Hp-1;
```

Comment:
thank you for your suggestions, they made sense and I added the changes + more scripts using percentheal to kill.